hail/src/main/scala/is/hail/expr/types/physical/PCanonicalCall.scala

```diff
@@ -1,10 +1,11 @@
 package is.hail.expr.types.physical
 
+import is.hail.annotations.Region
 import is.hail.asm4s._
 import is.hail.expr.ir.EmitFunctionBuilder
 import is.hail.variant.Genotype
 
-final case class PCanonicalCall(required: Boolean) extends PCall {
+final case class PCanonicalCall(required: Boolean = false) extends PCall {
```

Comment:
why this change?

Reply:
Because it looked like a mistake. All of our other concrete PTypes are either required = false by default on their only constructor, or have a required = false constructor.

This is the only case class that doesn't follow that pattern.